catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/integration/test/CatalogHiveIT.java

```diff
@@ -264,7 +264,7 @@ private static void createCatalog() {
   }
 
   private static void createSchema() throws TException, InterruptedException {
-    NameIdentifier ident = NameIdentifier.of(metalakeName, catalogName, schemaName);
+    NameIdentifier ident = NameIdentifier.of(schemaName);
```

Comment:
Why do we need to create a schema `ident`, from the code I think `schemaName` should be enough.

Reply:
Done